docs/populating-configuration.md

```diff
@@ -0,0 +1,353 @@
+* Run your own Keycloak server instance by following their [getting-started guide](https://www.keycloak.org/getting-started/getting-started-docker). Please note, how to setup Keycloak is out of scope of this guide
+
+To accomplish the previously mentioned functionalities Fleet Manager needs to
+be configured to interact with this additional SSO server.
+
+The SSO type to be used can be configured by setting the `--sso-provider-type` CLI flag when running
+Fleet Manager. The accepted values for it are `mas_sso` (keycloak is included here) or `redhat`.
+
+Create/Have available two client-id/client-secret pair of credentials (called Keycloak Clients)
+in the SSO Keycloak server to be used, one for each previously mentioned
+functionality, and then set Fleet Manager to use them by running
+the following command:
+```
+ SSO_CLIENT_ID="<sso-client-id>" \
+ SSO_CLIENT_SECRET="<sso-client-secret>" \
+ OSD_IDP_SSO_CLIENT_ID="<osd-idp-sso-client-id>" \
+ OSD_IDP_SSO_CLIENT_SECRET="<osd-idp-sso-client-secret>" \
+ make keycloak/setup
+```
+
+Additionally, if `redhat` is the configured SSO provider type then run
+the following to configure the credentials:
+```
+SSO_CLIENT_ID=<sso-client-id> \
+SSO_CLIENT_SECRET=<sso-client-secret> \
+make redhatsso/setup
+```
+
+Additionally, make sure you start the Fleet Manager server with the appropriate
+Keycloak SSO Realms and URL for this. See the
+[Fleet Manager CLI feature flags](./feature-flags.md#keycloak) for details on it.
+
+## Setup the data plane image pull secret
+In the Data Plane cluster, the Strimzi Operator and the FleetShard Deployments
+might reference container images that are located in authenticated container
+image registries.
+
+Fleet Manager can be configured to send this authenticated
+container image registry information as a K8s Secret in [`kubernetes.io/.dockerconfigjson` format](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials).
+
+In order for the Fleet Manager to be able to start, create the following file:
+```
+touch secrets/image-pull.dockerconfigjson
+```
+
+If you don't need to make use of this functionality you can skip this section.
+Otherwise, keep reading below.
+
+To configure the Fleet Manager with this authenticated registry information so
+the previously mentioned Data Plane elements can pull container images from it:
+* Base-64 encode your [Docker configuration file](https://docs.docker.com/engine/reference/commandline/cli/#docker-cli-configuration-file-configjson-properties).
+* Copy the contents generated from the previous point into the `secrets/image-pull.dockerconfigjson` file
+
+## Setup the Observability stack secrets
+Fleet Manager provides the ability to provide and configure monitoring metrics
+related to the Data Planes and its Kafka instances. To do so it makes
+use of what's called the ([Observability Stack](./observability/README.md#observability-stack)
```

Comment:
```suggestion
use of what's called the ([Observability Stack](./observability/README.md#observability-stack))
```

Reply:
The suggested change is not correct. The correct change is removing the initial parentheses.

I updated the text to do so.